crates/factors/tests/smoke.rs

```diff
@@ -8,6 +8,7 @@ use spin_factor_key_value::{KeyValueFactor, MakeKeyValueStore};
 use spin_factor_key_value_redis::RedisKeyValueStore;
 use spin_factor_outbound_http::OutboundHttpFactor;
 use spin_factor_outbound_networking::OutboundNetworkingFactor;
+use spin_factor_outbound_redis::OutboundRedisFactor;
```

Comment:
We've stopped adding new implementations to this smoke test, and I don't think it's necessary here.